src/components/Label/Label.stories.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import { Label } from './Label';
+
+export default {
+  title: 'UI Basic Elements/Label',
+  component: Label,
+} as ComponentMeta<typeof Label>;
+
+const Template: ComponentStory<typeof Label> = (args) => <Label {...args} />;
+
+export const MainLabel = Template.bind({});
```

Comment:
In the notion design doc for the frontend, let's reference the relevant documentation for storybook is possible